### ISSUE TITLE
[Web] Labels cell in collections table too big for large collections

### ISSUE THREAD

**Issue by dctrud** [singularityhub/sregistry#20]
Is there a limit on how big the 'labels' cell can get in the table on the 'collections' page? If you put a lot of images into a collection it seems to grow rather big. E.g. if I put all of the biocontainers (>1000) into a single collection would it be huge? Looks like there's a label for every distinct 'From:' in conatiners within the collection.

Not sure how would be best to deal with this.

![image](https://user-images.githubusercontent.com/4522799/29746599-68b27a10-8aa5-11e7-9a78-1b131ceccd43.png)

**Comment by vsoch** [comment]
I saw this! On the one hand, I was thrilled about all those labels, On the other hand, it's rather messy. Let me try writing a model function that returns shared labels across a collection , which might be more appropriate given being on that level?

**Comment by vsoch** [comment]
okay, I pushed an update that only shows the *shared* labels for the collection. This means you will see more for collections with fewer containers, fewer for collections with N=1 (or a small number):

![image](https://user-images.githubusercontent.com/814322/29746740-002aa708-8ab2-11e7-8b00-75190406edbb.png)

Also, I made the experience a bit more useful by providing a link to see containers associated with the label:

![image](https://user-images.githubusercontent.com/814322/29746744-187d5710-8ab2-11e7-8d32-4e6de01e04c2.png)


and you can navigate from there to see all the unique values for they key:

![image](https://user-images.githubusercontent.com/814322/29746746-2bded1c6-8ab2-11e7-8ccf-0587f757a137.png)


(bad example because there is only one value, "dinosaur" but given some other label of whatami "notdinosaur" you would see that row and associated containers.

**Comment by vsoch** [comment]
I'm being less filtery / conservative with the labels because I think a registry should generally be better at letting users search / find things.

**Comment by dctrud** [comment]
Looks good :+1:

**Comment by vsoch** [comment]
great!

**Comment by dctrud** [comment]
Not sure whether this is working as intended: https://sregistry.randomroad.net/collections

Can you confirm?

**Comment by vsoch** [comment]
Yeah it didn't look so busy for me! I first tried just removing it entirely, but this seems a bit devoid of information:

![image](https://user-images.githubusercontent.com/814322/29896994-02a63fbc-8dad-11e7-8057-a87368af7870.png)

It might be more useful to just show the unique container names in the collection, so I just did a push for that:

![image](https://user-images.githubusercontent.com/814322/29897760-75ea99de-8db0-11e7-99a0-c0f920df92a5.png)


clicking (right now) just does a general search across collections for the name, which could be improved with better search in the future. But I think this is a good approach to still give some meaning to the collections without having it be too much. What are your thoughts?
